### PR TITLE
CORE-15910 Configurable timeout for vNode registration check

### PR DESCRIPTION
**Pull request by marvincudjoe**
Make the timeout on polling for vNode registration configurable. The property to use is vnodeRegistrationTimeout

### CODE REVIEW

**Review by khutsijabari — APPROVED** [review]
LGTM

File: build.gradle [hunk 1]
```diff
@@ -33,6 +33,7 @@ allprojects {
         cordaCliBinDir = "${System.getProperty("user.home")}/.corda/cli"
         cpiUploadTimeout = cpiUploadDefault
         cordaProcessorTimeout = processorTimeout
+        vnodeRegistrationTimeout = vnodeRegistrationTimeoutDefault
```
Comment (yarivus) [inline build.gradle:36]: should we add a test for this new addition ?

**Review by yarivus — COMMENTED** [review]
see question/comment

File: build.gradle (hunk 1 above)
Comment (marvincudjoe) [reply build.gradle:36]: We'll be using the regpack for this. I'm looking to add more tests to the reg-pack once https://r3-cev.atlassian.net/browse/CORE-8176 is done (it's another configurable property task)

Comment (yarivus) [reply build.gradle:36]: thx for clarifying

**Review by yarivus — APPROVED** [review]
LGTM

**Pull request merged**